misc/ci/macos-aarch64-build-release.ini

```diff
@@ -0,0 +1,51 @@
+[constants]
+# apple silicon only supports MacOS 11.0, so just use that
+macos_min = '11.1'
+cflags = ['-mmacosx-version-min='+macos_min, '-arch', 'arm64']
```

Comment:
I think for most optimal results you should also pass `-mcpu=apple-m1`. If that doesn't work (e.g. due to LLVM version not being new enough), `-mcpu=apple-a14` can be used as it [seems to be basically identical](https://github.com/llvm/llvm-project/blob/61d1cce2f83571c00f76144d42a2dea2cb3ab1ca/llvm/include/llvm/Support/AArch64TargetParser.def#L195).

Reply:
`apple-m1` fails for some reason, so `apple-a14` it is!